python/cudf/cudf/_lib/parquet.pyx

```diff
@@ -467,6 +469,11 @@ cdef class ParquetWriter:
     max_page_size_rows: int, default 20000
         Maximum number of rows of each page of the output.
         By default, 20000 will be used.
+    nullability : bool, default None.
```

Comment:
The default is None but the behavior of the default value is not explained.

Reply:
Now not relevant with new changes.